### PR TITLE
Added RandAug and CutMix/Mixup (with label smoothing)

### PR DESCRIPTION
**Pull request by zaccharieramzi**

### CODE REVIEW

**Comment by zaccharieramzi** [comment]
I need to change the loss to BCE when using MixUp/CutMix. 

I will probably need to use https://pytorch.org/docs/stable/generated/torch.nn.BCEWithLogitsLoss.html , and be able to modify the `BenchPLModule`.
